sirepo/package_data/template/ml/scale.py.jinja

```diff
@@ -9,16 +10,16 @@ from sklearn.preprocessing import {{ dataFile_inputsScaler }}
 from sklearn.preprocessing import {{ dataFile_outputsScaler }}
 {% endif %}
 
-def read_data(path, **kwargs):
-    return sirepo.numpy.ndarray_from_csv(
-        path,
+def read_data(arch_mgr, data_path, **kwargs):
```

Comment:
archive_manager Avoid abbreviations unless they are necessary. In this case, arch_mgr is used once.

Reply:
[115e90b](https://github.com/radiasoft/sirepo/pull/4736/commits/115e90b0f6255ac567774bd06a36493f01dafe09)